src/Upgrader/Business/Upgrader/Validator/ReleaseGroupValidateManager.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Business\Upgrader\Validator;
+
+use Upgrader\Business\Command\Response\CommandResponse;
+use Upgrader\Business\DataProvider\Entity\ReleaseGroup;
+use Upgrader\Business\Exception\UpgraderException;
+
+class ReleaseGroupValidateManager implements ReleaseGroupValidateManagerInterface
```

Comment:
Please avoid Manager postfix.
Could it be ReleaseGroupValidator with isValid() method?

Reply:
renamed